common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -210,7 +246,7 @@ function (Sjson, AsyncProcess) {
             var self = this,
                 state = this.state,
                 language = state.getCurrentLanguage(),
-                data, youtubeId;
+                data, youtubeId, results, start, captions;
```

Comment:
Or alternatively, give them their own `var` statements:

```javascript
var results;
var start;
var captions;
```

I prefer this approach, but I'll defer to you.